Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+import SwiftUI
+
+public typealias SliderValue = BinaryFloatingPoint & Strideable & CustomStringConvertible
+
+/// Enables selection from a range of values.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/slider/)
+public struct Slider<Value>: View where Value: SliderValue, Value.Stride: BinaryFloatingPoint {
+
+    enum SelectedValue {
+        case single(Binding<Value>?)
+        case range(Binding<ClosedRange<Value>>?)
+    }
+
+    @Environment(\.sizeCategory) var sizeCategory
+
+    let value: SelectedValue
+    let valueLabel: String
+    let startLabel: String
+    let endLabel: String
+    let bounds: ClosedRange<Value>
+
+    @State private var isFirstThumbDrag = false
+    @State private var isSecondThumbDrag = false
+    @State private var firstValue: Value {
+        didSet {
+            switch value {
+                case .single(let currentValue):
+                    currentValue?.wrappedValue = firstValue
+                case .range:
+                    updateSelectedRangeIfNeeded()
+            }
+        }
+    }
+
+    @State private var secondValue: Value? {
+        didSet {
+            updateSelectedRangeIfNeeded()
+        }
+    }
+
+    @State private var dragStartOffset = 0.0
+
+    private let buckets: [Range<Value>]
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(valueLabel)
+                .foregroundColor(.blueNormal)
+                .padding(.bottom, .medium)
+                .accessibility(.sliderValueLabel)
+
+            SingleAxisGeometryReader(axis: .horizontal, alignment: .bottom) { width in
+                track
+                    .overlay(
+                        controls(sliderWidth: width)
+                    )
+                    .padding(.vertical, thumbVerticalOverlapping)
+            }
+
+            HStack {
+                Text(startLabel)
+                    .foregroundColor(.inkNormal)
+                Spacer()
+                Text(endLabel)
+                    .foregroundColor(.inkNormal)
+                    .multilineTextAlignment(.trailing)
+            }
+            .padding(.top, .xxSmall)
+            .accessibilityElement(children: .ignore)
+            .accessibility(label: .init(valueLabel))
+            .accessibility(hint: .init(accessibilityHint))
+        }
+    }
+
+    @ViewBuilder var track: some View {
+        RoundedRectangle(cornerRadius: 100)
```

Comment:
100? 🤔

Reply:
I've copied from Figma. I didn't want to introduce anything from my side even though 100 looks wrong
![image](https://user-images.githubusercontent.com/5494090/235730251-8edf9556-294c-492c-b166-833490c4c5cd.png)

Reply:
It does look wrong. A workaround or hack in figma does not necessarily translate to SwiftUI - both layouts have their own gotchas. Lets use a proper radius.